vanilla_installer/gtk/widget-language.ui

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <template class="LanguageRow" parent="AdwActionRow">
+    <child type="prefix">
+      <object class="GtkCheckButton" id="select_button"></object>
+    </child>
+    <property name="activatable-widget">select_button</property>
+    <child type="suffix">
+      <object class="GtkLabel" id="suffix_bin">
+        <property name="justify">center</property>
+        <property name="label" translatable="yes">Language</property>
+        <style>
+          <class name="monospace" />
+          <class name="dim-label" />
+      </style>
+      </object>
+  </child>
+  </template>
+</interface>
```

Comment:
```suggestion
</interface>

```

Reply:
The Contribution Guidelines do not include indication of wether the files should end with a new line or not. This suggestion should not be "PR based" but a global rule to be defined if we always want a new line at the end of the files.\
The definition should also state if the files must be LF, CRLF or if doesn't matter

Reply:
Agreed, we should probably specify that there must be an LF eol in the Contribution Guideline. I will send a PR for it soon. We already set Gitattributes for this in our CLI repos, will do the same for GUI ones.